net-proxy/haproxy/haproxy-1.6.9.ebuild

```diff
@@ -119,7 +123,6 @@ src_install() {
 	newconfd "${FILESDIR}/${PN}.confd" $PN
 	newinitd "${FILESDIR}/${PN}.initd-r3" $PN
 
-	dodoc CHANGELOG CONTRIBUTING MAINTAINERS
```

Comment:
> Fixed in last PR

It should be really ``einstalldocs`` here, unless you get the build
system to install the binaries in /usr/bin instead of /usr/sbin.